scripts/gan/cyclegan/README.md

```diff
@@ -0,0 +1,29 @@
+## A Mini Implementation Of Cyclagan
+
+
+**Prerequisites**
+1. Python 3.6+
+2. Gluoncv
+3. Mxnet
+
+
+**Download  Dataset**
+```bash
+cd ./datasets
+python extract_data.py
+```
+
+**Run The Model**
+```bash
+python Cyclegan.py --num_epoch 50 --lr 0.00001
+```
+
+**Visualize The Data**
+
+![sampleA](9_A.jpg "The Input A")
```

Comment:
If the images are only for display, you can drag them to any GitHub conversation, and right click and get a permanent link to the image without uploading to the repo.